webapp/src/view/appConfig/appConfig.tsx

```
@@ -17,54 +17,32 @@ import { Autocomplete, Box, Button, TextField, Typography, useTheme } from "@mui
-  const currentTheme = currentThemeConfig?.theme || Themes.DEFAULT_THEME;
+  const [setThemeQuery] = useSetThemeMutation();
+
+  if (!currentThemeConfig) {
+    return <ErrorHandler message={"Error retrieving theme"} />;
+  }
 
-  const [selectedTheme, setSelectedTheme] = useState<ThemeOption | null>(null);
+  const activeTheme = currentThemeConfig.activeThemeName;
+  const themeOptions = Object.values(currentThemeConfig.themes).map((theme) => theme.name);
 
   useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)